core/src/test/java/org/elasticsearch/action/update/UpdateRequestTests.java

```diff
@@ -49,7 +49,7 @@ public void testUpdateRequest() throws Exception {
         UpdateRequest request = new UpdateRequest("test", "type", "1");
         // simple script
         request.source(XContentFactory.jsonBuilder().startObject()
-                .field("script", "script1")
+                .startObject("script").field("inline", "script1").endObject()
```

Comment:
Good point, the test failed if I didn't make this change. I'll change the parsing this and other places so that a script string value is also accepted as an inline script.